js/customize-snapshots.js

```diff
@@ -1113,6 +1113,7 @@
 				isSameStatus = api.state( 'changesetStatus' ).get() === originalOptions.data.customize_changeset_status;
 				if ( 'customize_save' === originalOptions.data.action && isSameStatus && options.data ) {
 					options.data = removeParam( options.data, 'customize_changeset_status' );
```

Comment:
@westonruter Yes we wanted to auto save the date and title in editbox, but we could not do `wp.customize.previewer.save()` without status because the default status is `publish`, and we did not want to create revision each time it auto saves the date/title.